components/Arrivals.tsx

```diff
@@ -0,0 +1,40 @@
+import {useState} from 'react';
+import {getArrivals, Bus} from '../backend/fetchArrivals';
+import { StopCodeForm } from './StopCodeForm';
+import{BusCard} from './BusCard';
+
+export function Arrivals() {
+  const [arrivalsData, setArrivalsData] = useState<Bus[]>();
+  const [errorMessage, setErrorMessage] = useState<string>();
+
+  
+  async function handleGetArrivals(id: string){
+    let data = await getArrivals(id);
+    if( data.success){
+      setArrivalsData(data.array);
+    }
+    else{
+      setErrorMessage("No data found for stop code")
+    }
+  }
+
+  if (arrivalsData !== undefined){
```

Comment:
Might be cleaner to have one return statement and use gates to control what to include